app/views.py

```diff
@@ -252,8 +260,8 @@ def post(self, request: HttpRequest, submission_id: int) -> HttpResponse:
         transcription = transcription.replace("\r\n", "\n")
 
         issues = check_for_formatting_issues(transcription)
-        if len(issues) > 0:
-            # stash the important stuff in the session so that we can
+        if len(issues) > 0 or not transcription:
+            #            # stash the important stuff in the session so that we can
```

Comment:
Does `check_for_formatting_issues` work with `None`?

Reply:
We have a test for it now, so yes! :tada: